Sources/Hypertext.swift

```diff
@@ -49,8 +49,9 @@ extension Array: Renderable {
 }
 
 open class tag: Renderable {
-    public var name: String? = nil
-    public var isSelfClosing: Bool = false
+    open var isSelfClosing: Bool { return false }
```

Comment:
So I understand completely, the advantage of changing `isSelfClosing: Bool = false` to `isSelfClosing: Bool { return false }` is that a subclass of `tag` doesn't need to override the init method to change `isSelfClosing`, the subclass only needs to override the var, right?